gax-grpc/src/main/java/com/google/api/gax/grpc/RetryingCallable.java

```diff
@@ -32,25 +32,26 @@
 import com.google.api.gax.retrying.RetrySettings;
 import com.google.api.gax.retrying.RetryingExecutor;
 import com.google.api.gax.retrying.RetryingFuture;
+import com.google.api.gax.rpc.UnaryCallable;
 import com.google.common.base.Preconditions;
 
 /**
- * Implements the retry and timeout functionality used in {@link UnaryCallable}.
+ * Implements retry and timeout functionality.
```

Comment:
I went ahead and made this consistent, along with a couple missing `@BetaApi` annotations on public classes.